controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -32,13 +32,22 @@ def setUp(self):
              'target_vdisk_name': 'target_name',
              'id': 'test_fc_id',
              'status': FCMAP_STATUS_DONE,
-             'copy_rate': "non_zero_value"})]
+             'copy_rate': 'non_zero_value',
+             'rc_controlled': 'no'})]
+        self.fcmaps_as_target = [Munch(
```

Comment:
can you move all the fcmaps creations to a method so all the fcmap objects will be aligned and there will be less code duplication?

Reply:
https://github.com/IBM/ibm-block-csi-driver/pull/352#discussion_r689307391

Reply:
since this change adds duplications, IMO this suggestion is relevant to this PR

Reply:
Done